relay/contracts/BonsaiRelay.sol

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Risc0, Inc.
```

Comment:
Is it possible to use a single bonsai relay contract, I know the test contract has a few differences but would it make to sense to have the relay and test modes be able to use the same contract definition?

Reply:
I don't think they can be combined into a single contract. We need the cheatcodes in the test contract, and I don't think there's a convenient way to take those out pre-deployment.

Reply:
Ahh makes sense.